cacti/ci/cacti.rake

```diff
@@ -0,0 +1,57 @@
+require 'ci/common'
+
+def cacti_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def cacti_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/cacti_#{cacti_version}"
+end
+
+namespace :ci do
+  namespace :cacti do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
```

Comment:
From what I saw in the test, we're currently assuming there's a bunch of stuff available in the test box. That might not hold true, particularly in contributors systems. We have to ensure we provide everything required by the test, and then cleanup after ourselves. This test may not be ideally suited for docker (but we can try to make it work), otherwise we might have to bring some binaries into `embedded/` like we used to do with the `dd-agent` or build our own stuff. Let me know if you want to talk about it.